caprover/INSTALL_CAPROVER_ON_NEW_VM.md

```diff
@@ -0,0 +1,44 @@
+# New VM running CapRover
+
+This guide lists the high-level steps to get CapRover running on a new VM, so that you can then
+install the Guardian Connector stack.
+
+## Quick setup on popular cloud platforms
+
+We highly recommend using one of our quick guides that will handle all this for you
+in Azure or DigitalOcean — in which case **you may skip the rest of the document.**
+
+- To do this in Azure using an ARM template, see [`azure/README.md`](../azure/README.md)
+- To do this with a DigitalOcean Droplet one-click app, see [`digitalocean-vm/README.md`](../digitalocean-vm/README.md)
+
+## Do-it-yourself
+
+1. Create a VM on-premises or on the hosting provider of your choice.
+
+    > [!IMPORTANT]
+    > Your VM should be set up with sufficient CPU processing power, RAM, and disk storage.
+    >
+    > Our current default for the Guardian Connector stack is a VM with 2 vCPUs, 4 GB Memory, and a 35 GB Disk.
+    >
+    > Anything less than that may run into performance issues, or the applications may not start at all.
```

Comment:
Kind of annoying that this doesn't render on Github; I guess because of the indent. But it does in other environments like using VSCode Markdown preview. Not important.

Reply:
thanks for discovering this. I think it's important to render on GitHub so I will be changing the indentation.